package.json

```diff
@@ -37,7 +37,7 @@
     "url": "git+https://github.com/CraveFood/farmblocks.git"
   },
   "devDependencies": {
-    "@crave/farmblocks-tooltip": "^0.2.0-alpha.e320de4a",
+    "@crave/farmblocks-tooltip": "^0.3.0",
```

Comment:
I think it is safe to remove this line as well, since tooltip is on the workspaces already